protos/service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package tasktimer;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/nivista/tasktimer/.gen;pb";
+
+service TaskTimer {
+    rpc CreateTimer (CreateTimerRequest) returns (CreateTimerResponse) {}
+
+    rpc DeleteTimer (DeleteTimerRequest) returns (DeleteTimerResponse) {}
+
+    rpc GetTimer (GetTimerRequest) returns (GetTimerResponse) {}
+}
+
+message CreateTimerRequest {
+    string account = 1;
+    TaskConfig task_config = 2;
+    ScheduleConfig schedule_config = 3;
+}
+
+message CreateTimerResponse {
+    string id = 1;
+}
+
+message DeleteTimerRequest {
+    string account = 1;
+    string id = 2;
+}
+
+message DeleteTimerResponse {}
```

Comment:
There are two types of successful deletes: timer did not exist and you tried delete, timer existed and you deleted it. Its possible to count the first one as an error or its possible to include some result in this struct that contains which of those two cases it was.